e2e/testdata/fn-render/path-index-duplicate/starlark-httpbin-gen.yaml

```diff
@@ -0,0 +1,61 @@
+# Copyright 2021 Google LLC
```

Comment:
Can we rename this file to `starlark-gen-duplicate-path.yaml` to make it obvious.